docs/src/_tutorials/jupyter.md

```diff
@@ -5,7 +5,7 @@ layout: doc
 weight: 4
 ---
 
-As of _Sep-2022_, jupyter is not yet available as utility on the [hub](https://hub.meltano.com/), so you need to add it to your Meltano project as (local) custom plugin. Even if it is available as plugin on the hub, you will probably want to do some customization, so it will make sense to create your own version.
+As of _Sep-2022_, jupyter is not yet available as utility on the [Meltano Hub](https://hub.meltano.com/), so you need to add it to your Meltano project as (local) custom plugin. Even if it is available as plugin on the Hub, you will probably want to do some customization, so it will make sense to create your own version.
```

Comment:
```suggestion
As of _Sep-2022_, jupyter is not yet available as utility on [Meltano Hub](https://hub.meltano.com/), so you need to add it to your Meltano project as (local) custom plugin. Even if it is available as plugin on the Hub, you will probably want to do some customization, so it will make sense to create your own version.
```

We have a style guide for the Product family (though it seems like there's a rendering issue on the page currently. https://handbook.meltano.com/marketing/style-guide#how-to-talk-about-the-meltano-product-family-and-team